views/_alert.php

```diff
@@ -16,14 +16,17 @@
  */
 ?>
 
-<?php if ($module->enableFlashMessages): ?>
+<?php if ($module->enableFlashMessages) : ?>
     <div class="row">
         <div class="col-xs-12">
-            <?php foreach (Yii::$app->session->getAllFlashes() as $type => $message): ?>
-                <?php if (in_array($type, ['success', 'danger', 'warning', 'info'])): ?>
-                    <?= Alert::widget(['options' => ['class' => 'alert-dismissible alert-'.$type], 'body' => $message]) ?>
+            <?php foreach (Yii::$app->session->getAllFlashes() as $type => $message) : ?>
+                <?php if (in_array($type, ['success', 'danger', 'warning', 'info'])) : ?>
+                    <?= Alert::widget([
+                            'options' => ['class' => 'alert-dismissible alert-'.$type],
+                            'body' => $message
+                    ]) ?>
                 <?php endif ?>
             <?php endforeach ?>
         </div>
     </div>
-<?php endif ?>
```

Comment:
All other php files do not end with a `?>` so I thought view files should also not end with a `?>`.
See commits below, I have changed it for you :)